ios/Podfile.lock

```diff
@@ -551,4 +551,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 7f4c21e9567b9edf7b64e41ed7613898a792497c
 
-COCOAPODS: 1.7.5
+COCOAPODS: 1.9.1
```

Comment:
@dereckquock won't cocoa pods updates require a new build in the App Store?

Reply:
i'm not sure 🤔 but lemme just revert this for now